ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_peer_groups.schema.yml

```diff
@@ -164,6 +183,25 @@ keys:
           bfd:
             type: bool
             default: False
```

Comment:
@gmuloc is this correct? I would have thought we always wanted BFD on this.

Reply:
The defaults what we have used in the modules for bfd is actually true by default for most of the peers. ( used in the file bgp_peer_groups.py )
Not sure if this default overrides the default used in the file "bgp_peer_groups".